package.json

```diff
@@ -93,6 +94,7 @@
         "clean": "hereby clean",
         "gulp": "hereby",
         "lint": "hereby lint",
+        "knip": "knip --tags=-internal,-knipignore --exclude duplicates",
```

Comment:
Just to note it, I would instead have this as a task in the `Herebyfile`, which is more typical in this codebase.